### DISCUSSION TITLE
Western Pennsylvania Mesh

### DISCUSSION THREAD

**Discussion by rjDipcord** [meshtastic/Discussions#136] [North America]
Hello everyone,

I am coordinating the development of a mesh in western Pennsylvania. At the time of writing, it is starting to stretch outward from Westmoreland county. I've installed 4 routers on prominent tower sites I have access to, and gotten a good portion of my amateur radio community (and nerds) involved. 

I'm interested in expanding and collaborating with others who wish to do the same, and improve the mesh in our area. This is just one way we can discuss more. Please reach out here if you're interested too!

**Comment by carols10cents** [comment]
👋🏻 Hi, I'm in Allegheny County, just got 2 nodes for Christmas and know next to nothing but I'm excited to get connected with the mesh!

**Reply by rjDipcord** [reply]
As of now, there is one node I see pop up when I am downtown. I am not sure their position though. Then there are a few nodes in McKeesport that are elevated enough to see my router in Greensburg. I have things in the works to get one operational in New Kensington and Apollo. If you see a node with "KC3IJL" in the name, say Hi!

**Reply by kabili207** [reply]
@rjDipcord Ooh~ I've seen you pass by my house near the turnpike a few times

**Reply by rjDipcord** [reply]
> @rjDipcord Ooh~ I've seen you pass by my house near the turnpike a few times

Hi! 👋

You may have seen me send a "beep beep" a few times!

Are you in the PghMesh discord?

**Reply by kabili207** [reply]
Nope, didn't know we had one. Got a link?

**Reply by rjDipcord** [reply]
> Nope, didn't know we had one. Got a link?

Come join the fun! https://discord.com/invite/zbXzBf84vR

**Comment by bryanpendleton** [comment]
I've got a smattering of nodes, centered around the middle of Pittsburgh (Sq Hill South). Have been developing a solar node I was hoping to get put up high on a building within the city, though I let that stall enough that I'll wait out the freezing temps before I push again.

**Reply by rjDipcord** [reply]
If you have friends at the steel tower, It is visible from my router in Apollo. That is, when I am 200ft up the tower, I can see it clearly. I will do a propagation report and see if there are any other buildings that work. Because getting up high in Pgh would be tremendous.

**Reply by rjDipcord** [reply]
This is promising! My router in Apollo can see a LOT of downtown, assuming another node was on top of a building.

In this scan, we can see much of downtown is yellow. This report was run using a worst-case scenario with an assumed antenna height (downtown) at 50m. Considering many of the buildings downtown are taller than that, I think 50m was a good middle ground.
![image](https://github.com/user-attachments/assets/17d5ae70-b8d4-48ad-b96a-15bf1b58507d)

This is good. It means we basically don't have to consider which building is used at all. As long as they're not too close to Point Park, we get good signal. With that said, however. I think it would be much more beneficial to get a node on Children's Hospital. It has good coverage as well, and can be seen by Apollo.

Then again, I might have access to a tower in Hazelwood that seems to get good coverage in the greater PGH area. Here's that report:
![image](https://github.com/user-attachments/assets/cce34a98-e503-4809-839b-780e5ccb7315)
I think this one will be on my radar for sure. I can probably get it established spring time 2025.

**Reply by bryanpendleton** [reply]
Children's hospital is actually where I'd been hoping to put up my solar node. There's already an amateur radio repeater there, run by a neighbor. A tower in Hazelwood would also be pretty great.

We should probably find an alternate channel for coordination.

**Reply by carols10cents** [reply]
I'm in Stanton Heights at the top of the hill-- I can see across the river to rt 28 modulo trees, and there's a chance I might be able to get line of sight to the steel tower from my roof (I think I need a better antenna though).

I've got one node I've been leaving at home and one I've taken out and about; I've seen a few nodes downtown and one that looks like it's at the CMU robotics center around 45th st in Lawrenceville.

**Comment by rjDipcord** [comment]
I've been brewing some more updates to the WPA mesh recently. I did a brief [write-up on my website](https://kc3ijl.site/new-meshtastic-routers-apollo-and-pine-ridge/) about them. One more high-profile router is now part of our infrastructure!

**Comment by gdfox45-hub** [comment]
Good morning!!
I work in South side and live in Delmont. I’m new to Meshtastic and I don’t have any amateur radio experience. I still need to buy two nodes for my wife and I and a solar to bounce off of somewhere close to Delmont. 
any recommendations or advice would be greatly appreciated. 
thanks in advance. 
-Greg

**Reply by rjDipcord** [reply]
Hey Greg! I encourage you to visit the WPAMesh website at https://WPAmesh.net

Lots of information there, as well as links to our various social media outlets.

**Reply by gdfox45-hub** [reply]
Thank you RJ😁
